### ISSUE TITLE
[TF2] [Feature Request] Implement a bi-weekly map cycle for Core game modes in Casual mode.

### ISSUE THREAD

**Issue by gaelcoral** [ValveSoftware/Source-1-Games#5426]
In Casual mode, the number of maps continues increasing. This could be seen as positive, since it offers more variety. However, this increase has resulted in certain maps being forgotten or filled with "cheaters bots", especially in less popular regions. This is particularly noticeable in game modes such as Control Points and Payload, which have 18 maps each, leading to the Matchmaking system ending up offering the same two maps.

I suggest implementing a map cycle that updates bi-weekly (every two weeks), which would help revive interest in forgotten maps and reduce queue time in less popular regions.

From my opinion, each game mode should have a cycle of 8 maps. For example, in Payload instead of having 18 maps, it would now have 8.
### Maps that should be permanently available in the map cycles:

- The official maps created by Valve
- And Maps that are required to complete certain contracts (such as Mercenary Park)

This suggestion would also allow more maps to be included in future updates, without worrying about map saturation. 

**Here you can see that there are currently 70 maps in Core game mode in Casual mode**
![copypate](https://github.com/ValveSoftware/Source-1-Games/assets/63524243/25af7d7f-edea-419c-b661-c62095e03911)
**If my suggestion is implemented, this would be reduced to 39 maps, a 44% decrease!**
![copypate](https://github.com/ValveSoftware/Source-1-Games/assets/63524243/94900983-b531-4205-9249-26db56a510a7)

**Comment by Kruphixx** [comment]
I agree that something should be done eventually especially with the amount of maps in Casual increasing each year. 

I'd be fine with your solution but curious if anyone else got some other ideas.

**Comment by gaelcoral** [comment]
now that I think about it, I think weekly is much better than biweekly

**Comment by gaelcoral** [comment]
Now with the Summer 2025 update, this issue has become much more noticeable
